docs/UserGuide.md

```diff
@@ -195,197 +245,166 @@ Examples:
 
 Example command usages from the current student list shown in Figure 1:
 
-* `find_student John` returns John Lee
+* `find_student John` returns "John Lee"
 * `find_student Sam` returns nothing
 * `find_student Lee` returns "John Lee" and "Samuel Lee"
-* `find_student Johnz Lee` returns "Johnz Tan", "John Lee" and "Samuel Lee"
+* `find_student Johnz Lee` returns "John Lee", "Johnz Tan" and "Samuel Lee"
 
-#### Adding a student: `add_student`
+#### Deleting a student: `delete_student`
 
-Adds a student to the TutorBuddy
+Deletes the specified student from TutorBuddy.
 
-Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN` <br>
-* `STUDENT_PHONE_NUMBER`, `GUARDIAN_PHONE_NUMBER` should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and 8 digits)
+Format: `delete_student STUDENT_INDEX` <br>
+
+<div markdown="span" class="alert alert-primary">:bulb: Tip:
+Use `find_student` command to filter the list of students you want.<br>
+The STUDENT_INDEX will be based on the current list.<br>
+(E.g. If John Doe is the 500th student, instead of `delete_student 500`, you can use `find_student John Doe` and
+`delete_student 1` if the John Doe you would like to delete is the 1st in the list)
+</div>
 
 Examples:
-* `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, Block 123, #01-01 l/Sec2 g/95421323 r/Mother`
+* `delete_student 2` deletes the 2nd student based on the current list of students.
 
 #### Editing a student: `edit_student`
```

Comment:
Just thinking that edit_student before delete_student might be a better flow. Since a user can add a student, edit that student and then proceed to remove it after.

Reply:
ok!